webapp/components/signup_user_complete.jsx

```diff
@@ -295,22 +295,6 @@ export default class SignupUserComplete extends React.Component {
         }
 
         const providedPassword = ReactDOM.findDOMNode(this.refs.password).value;
-        if (!providedPassword || providedPassword.length < Constants.MIN_PASSWORD_LENGTH) {
```

Comment:
Maybe re-add the clientside validation here? You've got the function in utils.jsx that checks the password so you could check if it returns null or not.